secrets-provider-test-chart/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
+name: secrets-provider-test-chart
+version: 1.0.0
```

Comment:
this is the `secrets-provider` version; should be `1.0`, no?

Reply:
Actually, this is the version of the chart; `appVersion` is the version of the `secrets provider`.
But it should be the same, I think.